pytorch_lightning/metrics/__init__.py

```diff
@@ -30,3 +30,11 @@
     PSNR,
     SSIM,
 )
+
+from pytorch_lightning.metrics.retrieval import (
```

Comment:
I would recommend that all metrics are renamed such that it is clear that they have something to do with information retrieval. For example
`MeanAveragePrecision`->`RetrivalMAP` or
`MeanAveragePrecision`->`IRMeanAveragePrecision`